clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/RecoveryIntegrationSpec.scala

```diff
@@ -28,59 +39,68 @@ class RecoveryIntegrationSpec
   private def updateDoc(json: Json, pathFieldName: String): Json = {
     val oldUri = json.hcursor.get[String](pathFieldName).fold(throw _, identity)
     val updates = Seq(
-      ClioDocument.UpsertIdElasticSearchName -> Json.fromString(UpsertId.nextId().id),
-      pathFieldName -> Json.fromString(s"$oldUri/$randomId")
+      ElasticsearchIndex.UpsertIdElasticsearchName -> UpsertId.nextId().asJson,
+      pathFieldName -> s"$oldUri/$randomId".asJson
     )
     json.deepMerge(Json.fromFields(updates))
   }
 
+  private val ubamMapper =
+    ElasticsearchDocumentMapper[TransferUbamV1Key, TransferUbamV1Metadata]
```

Comment:
Looks like this is another case where we could refactor to reduce code duplication, in a future PR.